src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx

```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
 
 You can also [extend your agent instrumentation](#extend-instrumentation) and explore your data with other New Relic products, including New Relic One's [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
+## Evaluate our Java agent with guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the Java agent. 
+
+<Callout variant="caution">
+The guided install version of the Java agent is for evaluation purposes only. It will not persist when you restart your Java application.
+</Callout> 
+
+![A screenshot of the guided install CLI.](./images/guided-install-cli.png "The guided install CLI.")
+
+Ready to get started? Click one of these buttons to try it out.
+
+<ButtonGroup>
+<ButtonLink
+  role="button"
+  to="https://one.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJhcG0iLCJsYW5ndWFnZSI6ICJqYXZhIiwgInJlY2lwZU5hbWUiOiAiamF2YS1hZ2VudC1pbnN0YWxsZXIiLCAiaW5pdGlhbEFjdGlvbkluZGV4IjoyLCJhY2NvdW50SWQiOm51bGx9"
+  variant="primary"
+>
+  Guided install
+</ButtonLink>
+
+<ButtonLink
+  role="button"
+  to="https://one.eu.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJhcG0iLCJsYW5ndWFnZSI6ICJqYXZhIiwgInJlY2lwZU5hbWUiOiAiamF2YS1hZ2VudC1pbnN0YWxsZXIiLCAiaW5pdGlhbEFjdGlvbkluZGV4IjoyLCJhY2NvdW50SWQiOm51bGx9"
+  variant="primary"
+>
+  EU Guided install
+</ButtonLink>
+</ButtonGroup>
+
+In addition to the Java agent, you can use this to install our infrastructure agent to discover other applications, infrastructure, and log sources running in your environment and recommend which ones should be instrumented. The install automates the configuration and deployment of each system you choose to instrument.
```

Comment:
So the Java agent guided install doesn't completely install the Java agent, but clicking those buttons will still let me install Infra and other things? (Not necessarily a critique of the writing here, just making sure I understand what all can happen when I click the button.)

Reply:
Well, it does completely install the Java agent. It's just that the Java agent installation isn't persistent. When you restart your Java application, it effectively breaks your application's connection with the agent. This means that you'll no longer be sending data to New Relic until you restart the agent too.